projects/core/constants/regexp.ts

```diff
@@ -2,6 +2,7 @@ export const TUI_DIGIT_REGEXP = /\d/;
 export const TUI_NON_DIGIT_REGEXP = /\D/;
 export const TUI_NON_DIGITS_REGEXP = /\D+/g;
 export const TUI_LEADING_ZEROES_REGEXP = /^0+/;
+export const TUI_NON_PLUS_AND_DIGITS_REGEX = /[ \-_()]/g;
```

Comment:
```suggestion
export const TUI_NON_PLUS_AND_DIGITS_REGEX = /[^\d\+]/g;
```

Reply:
everything not digits and not plus

Reply:
P.S. i am not master of reg exps.
Please double check me :)

Reply:
<img width="322" alt="image" src="https://user-images.githubusercontent.com/12021443/160667810-10b45f78-d798-4c3b-8e82-aa34c28ac4f4.png">

Reply:
ok, I think naming is wrong. I moved regexp from input-phone, it uses to clear value from brackets, dashes and underscores. How about TUI_MASK_SYMBOLS_REGEX?